src/draw_buffer.rs

```diff
@@ -35,13 +37,11 @@ impl ColorAttachment {
             ColorAttachment::Texture(_) => ColorAttachmentType::Texture,
         }
     }
-}
 
-impl Drop for ColorAttachment {
-    fn drop(&mut self) {
-        match *self {
-            ColorAttachment::Renderbuffer(id) => gl::delete_renderbuffers(&[id]),
-            ColorAttachment::Texture(tex_id) => gl::delete_textures(&[tex_id]),
+    fn destroy(self, gl: &gl::Gl) {
```

Comment:
Didn't want to grow them by an extra pointer given we only create one per draw buffer, but I can definitely do it I guess.